source/sass/component/_acceptance.scss

```diff
@@ -81,6 +85,13 @@ $c-acceptance-background-color: var(--c-acceptance-background-color, $color-back
             left:50%;
             transform: translate(-50%, -50%);
             color: $c-acceptance-alpha-color-contransting;
+            transition: text-shadow 0.1s linear;
+            text-shadow: 0 0 30px rgba(0,0,0,0.3);
+
+            &:hover {
+                opacity: 1;
+                text-shadow: 0 0 10px rgba(0,0,0,0.5);
```

Comment:
This may be interfering with other design decisions (visually). Consider using one of the shadow level vars.

Reply:
Also: Try using a calculation of $base for any sizes (8px).